mod-source-record-manager-server/src/main/java/org/folio/services/flowcontrol/RawRecordsFlowControlServiceImpl.java

```diff
@@ -59,7 +59,7 @@ public void init() {
    * threshold can be missed during resetting that can cause that resume/pause cycle may not be as usual, because we are
    * starting from clear state after reset, but any events would not be missed and consumers never pause forever.
    */
-  @Scheduled(cron = "${di.flow.control.reset.state.cron:0 0/5 * * * ?}")
+  @Scheduled(fixedRateString = "${di.flow.control.reset.state.interval:PT5M}")
```

Comment:
I made this change, because its hard to setup cron expression in rancher, cron is cutting of in rancher, so basically this style is more convenient, now this job also triggered each 5 mins.
Here is example of syntacs to specifying values, its Spring based Duration class:
![image](https://user-images.githubusercontent.com/25097693/170243697-67224718-1b50-49d1-bc3a-9adcc9588085.png)